charts/managed-identity-wallets/values-int.yaml

```diff
@@ -14,6 +14,23 @@ datapool:
 revocationService:
   baseUrlForCredentialList: "https://managed-identity-wallets.int.demo.catena-x.net/api/credentials/"
   clientIssuanceApiUrl: "http://localhost:8080"
+# -- The logging configurations
+logging:
+  exposed: "OFF"
+# -- The timeout (milliseconds) configuration of http client in each service.
+httpClientTimeout:
+  walletServiceRequest: "60000"
```

Comment:
I believe increasing the timeout is not the solution, if something is blocking it only blocks longer and makes the problem worse. So I would keep the default of 30 seconds or even reduce it.